config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   get 'books/index'
+  get 'books/new'
+  resources :books
```

Comment:
Because you've declared `resources :books`, you don't need the two `get`s above; Ruby automatically declares them for you